### PR TITLE
Correct missing docstrings, and update lint configuration

### PR DESCRIPTION
**Pull request by janiversen**
First commit:
- solve black formatting problem in command.py.
Second commit:
- Activate D100-D107 for flake8.
- Disable "locally-disabled" (as recommended in the previous PR.
Third commit:
- Add doc strings

### CODE REVIEW

File: tests/api/test_aiocoap_api.py [hunk 1]
```diff
@@ -17,40 +20,56 @@ def wrapper(*args, **kwargs):
 
 
 class MockCode:
+    """Mock Code."""
+
     def is_successful(self):
+        """Is succesful."""
```
Comment (MartinHjelmare) [inline tests/api/test_aiocoap_api.py:26]:
```suggestion
        """Is successful."""
```

File: tests/moods.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#  Retrieved from Gateway running on 1.9.27
+"""MOOD definitions: retrieved from Gateway running on 1.9.27"""
```
Comment (MartinHjelmare) [inline tests/moods.py:1]:
```suggestion
"""Mood definitions: retrieved from Gateway running on 1.9.27"""
```

File: tests/test_device.py [hunk 3]
```diff
@@ -502,6 +529,7 @@ def test_deviceinfo_battery_level(comment, device):
 
 @pytest.mark.parametrize(*input_devices)
 def test_deviceinfo_battery_level_unkown(comment, device):
+    """Rest battery level unknown."""
```
Comment (MartinHjelmare) [inline tests/test_device.py:532]:
```suggestion
    """Test battery level unknown."""
```

**Comment by janiversen** [comment]
Committed all your suggestions.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**